pegasus/sites.v3/code.org/public/js/admins_email_signup_form.js

```diff
@@ -0,0 +1,17 @@
+// Code provided from Google reCAPTCHA
+function timestamp() { var response = document.getElementById("g-recaptcha-response"); if (response == null || response.value.trim() == "") {var elems = JSON.parse(document.getElementsByName("captcha_settings")[0].value);elems["ts"] = JSON.stringify(new Date().getTime());document.getElementsByName("captcha_settings")[0].value = JSON.stringify(elems); } } setInterval(timestamp, 500);
+
+setInterval(timestamp, 500);
+
+// Set a disabled attribute on button until
+// the reCAPTCHA checkbox is clicked
+function recaptchaCallback() {
+  const btnSubmit = document.querySelector("button");
+
+  if (btnSubmit.hasAttribute("disabled")) {
+    btnSubmit.removeAttribute("disabled");
+    btnSubmit.setAttribute("disabled");
```

Comment:
I'm confused about this –– we remove the attribute and then set it too?

Reply:
Oh yeah, we don't need to set it too! I adapted this from [this Stackoverflow answer](https://stackoverflow.com/a/40881272) and did a 1:1 swap w/ the attribute methods bc my JS skills are quite rusty 😅 🤦🏻‍♀️ I'll remove the .setAttribute() method now!